pkg/resources/management.cattle.io/v3/globalrolebinding/validator.go

```diff
@@ -119,12 +132,29 @@ func (a *admitter) Admit(request *admission.Request) (*admissionv1.AdmissionResp
 	}
 	err = auth.ConfirmNoEscalation(request, clusterRules, "", a.grbResolver)
 	if err != nil {
+		hasBind, bindErr := auth.RequestUserHasVerb(request, globalRoleGvr, a.sar, bindVerb, globalRole.Name, "")
+		if bindErr != nil {
+			logrus.Warnf("Failed to check for the 'bind' verb on GlobalRoles: %v", bindErr)
+			return admission.ResponseFailedEscalation(err.Error()), nil
+		}
+		if hasBind {
+			// if we have the bind permission, no need to evaluate permissions further
+			return admission.ResponseAllowed(), nil
+		}
 		return admission.ResponseFailedEscalation(err.Error()), nil
 	}
 
 	rules := a.grbResolver.GlobalRoleResolver.GlobalRulesFromRole(globalRole)
 	err = auth.ConfirmNoEscalation(request, rules, "", a.resolver)
 	if err != nil {
+		hasBind, bindErr := auth.RequestUserHasVerb(request, globalRoleGvr, a.sar, bindVerb, globalRole.Name, "")
+		if bindErr != nil {
+			logrus.Warnf("Failed to check for the 'bind' verb on GlobalRoles: %v", bindErr)
+			return admission.ResponseFailedEscalation(err.Error()), nil
+		}
+		if hasBind {
+			return admission.ResponseAllowed(), nil
+		}
 		return admission.ResponseFailedEscalation(err.Error()), nil
 	}
```

Comment:
nit: this could potentially be extracted as a separate `func`, and used here and at `133`-`145`.